stubs/_blueman.pyi

```diff
@@ -0,0 +1,58 @@
+from typing import Any, List, Dict, Union, Callable
+from mypy_extensions import TypedDict
```

Comment:
Should be `typing_extensions` to stay consistent.

Reply:
Will fix